### ISSUE TITLE
It is necessary to create a universal patch for adapt modern open source software under Windows XP/2003

### ISSUE THREAD

**Issue by EgorKuzevanov** [Skulltrail192/One-Core-Api#34]
Hello! Can you create a universal patch for adapt modern open source software under Windows XP/2003? The fact is that even open source software developers lose huge amounts of money due to long-term support for Windows XP and, in order to earn more money, deliberately tie their developments to modern versions of Windows, despite the fact that their software can theoretically work on both newer and on older versions of Windows.

You need to adapt the following software:
- Mozilla Firefox 53+ (installers and programs itself)
- Mozilla Thunderbird 53+ (installers and programs itself)
- SeaMonkey 2.53+ (installers and programs itself)
- Pale Moon 27+ (installers and programs itself)
- Chromium 50+ (installers and programs itself)
- LibreOffice 6.x & LibreOffice 7.x (installers and programs itself)
- Inkscape 1.x (installers and programs itself)
- MuseScore 2.x & MuseScore 3.x (installers and programs itself)
- Calibre 3.x (installers and programs itself)
- Open Broadcaster Software & OBS Studio (installers and programs itself)
- Godot Game Engine
- FontForge 2020-11-07 (installer and program itself)
- FFmpeg (versions released after 2016-2017)
- GnuCash 3.x (installers and programs itself)
- Audacity 2.4.x (installers and programs itself)
- Apache HTTP Server 2.4.x

Thanks in advance.

**Comment by mina354** [comment]
You're completely right, but I would say the following need to have a universal patch first:

- Mozilla Firefox 53+ (installers and programs itself)
- Mozilla Thunderbird 53+ (installers and programs itself)
- SeaMonkey 2.53+ (installers and programs itself)
- Chromium 50+ (installers and programs itself)
- LibreOffice 6.x & LibreOffice 7.x (installers and programs itself)
